controllers/food-controllers.js

```diff
@@ -71,7 +72,8 @@ const addMealOption = async (req, res, next) => {
 
 const addMeal = async (req, res, next) => {
   try {
-    const { email } = req.session.user;
+    const { email } = await User.findOne({ _id: req.userData.userId });
```

Comment:
I see this line of code in every method, wont it be better to add it to the checkAuth method?